pom.xml

```diff
@@ -168,7 +168,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>2.1.16</version>
+            <version>2.3.19</version>
```

Comment:
![image](https://user-images.githubusercontent.com/88530400/141259496-4e40f52a-f9ea-4563-8aa6-5139a0fd9f71.png)

Just wanted to know why are we using 2.3.19, instead of this one can we use 2.6.1 as it has more installs and recent one. Upgrading to the recent one will also remove the fossa scan reported vulnerabilities.
cc : @rishi-104276